### ISSUE TITLE
DL3047 is triggered when using docker ARG

### ISSUE THREAD

**Issue by lit-af** [hadolint/hadolint#970]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
No issue should be detected

### Actual behavior
The issue is detected on every line there's a wget command

### Steps to reproduce the behavior
Run hadolint on the dockerfile

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 2.12.0
```

**Dockerfile** (if relevant)

```Dockerfile
FROM ubuntu:22.04

ARG CMAKE_VERSION=3.26.4
ARG WGET_ARGS="--progress=dot:giga"

#Install CMake
RUN wget ${WGET_ARGS} "https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-linux-x86_64.tar.gz" -O cmake-${CMAKE_VERSION}-linux-x86_64.tar.gz \
    && tar -zxf cmake-${CMAKE_VERSION}-linux-x86_64.tar.gz \
    && rm cmake-${CMAKE_VERSION}-linux-x86_64.tar.gz \
    && mv cmake-${CMAKE_VERSION}-linux-x86_64 /cmake-tool
ENV PATH="/cmake-tool/bin:${PATH}"
```

**Additional environment details (OS, stack version, etc.)**
